source/team/operational-processes.html.md.erb

```diff
@@ -35,6 +35,54 @@ The whole support team will:
 
 Before ending the day (4PM), an engineer will hand over information about planned work or high-priority incidents to the on-call engineer.
 
+## Out-of-Hours Planned Changes Support
+
+This process ensures that planned maintenance and updates are executed smoothly with appropriate support during out-of-hours periods.
+
+### Requesting Out-of-Hours Support
+
+#### Advance Notice
+
+Teams must provide a minimum of 48 hours notice for any planned changes requiring out-of-hours support.
+
+#### Submission Process
+
+Submit requests by creating a GitHub issue using the provided template. Use this link to access the template: [Insert GitHub Issue Template Link When Created]
+
+Once complete please notify the team via the [#ask-modernisation-platform](https://mojdt.slack.com/archives/C01A7QK5VM1) Slack channel once the issue is created.
+
+#### Information to Include
+
+Change Details: A comprehensive description of the planned change.
```

Comment:
Am I right in thinking that this needs more spacing or bullet pointing for line separation?

Reply:
Ah yes copying from docs i missed a few things